charts/private-action-runner/values.yaml

```diff
@@ -13,104 +13,74 @@ runners:
-        # - com.datadoghq.kubernetes.apps.getDaemonSet
-        # - com.datadoghq.kubernetes.apps.getDeployment
-        # - com.datadoghq.kubernetes.apps.getReplicaSet
-        # - com.datadoghq.kubernetes.apps.getStatefulSet
-        # - com.datadoghq.kubernetes.apps.listControllerRevision
-        # - com.datadoghq.kubernetes.apps.listDaemonSet
-        # - com.datadoghq.kubernetes.apps.listDeployment
-        # - com.datadoghq.kubernetes.apps.listReplicaSet
-        # - com.datadoghq.kubernetes.apps.listStatefulSet
-        # - com.datadoghq.kubernetes.apps.restartDeployment
-        # - com.datadoghq.kubernetes.apps.updateDeployment
-        # - com.datadoghq.kubernetes.apps.updateReplicaSet
-        # - com.datadoghq.kubernetes.core.createNode
-        # - com.datadoghq.kubernetes.core.createPod
-        # - com.datadoghq.kubernetes.core.deleteMultiplePods
-        # - com.datadoghq.kubernetes.core.deletePod
-        # - com.datadoghq.kubernetes.core.getConfigMap
-        # - com.datadoghq.kubernetes.core.getEndpoints
-        # - com.datadoghq.kubernetes.core.getEvent
-        # - com.datadoghq.kubernetes.core.getLimitRange
-        # - com.datadoghq.kubernetes.core.getNamespace
-        # - com.datadoghq.kubernetes.core.getNode
-        # - com.datadoghq.kubernetes.core.getPersistentVolume
-        # - com.datadoghq.kubernetes.core.getPersistentVolumeClaim
-        # - com.datadoghq.kubernetes.core.getPod
-        # - com.datadoghq.kubernetes.core.getPodTemplate
-        # - com.datadoghq.kubernetes.core.getReplicationController
-        # - com.datadoghq.kubernetes.core.getResourceQuota
-        # - com.datadoghq.kubernetes.core.getService
-        # - com.datadoghq.kubernetes.core.getServiceAccount
-        # - com.datadoghq.kubernetes.core.listConfigMap
-        # - com.datadoghq.kubernetes.core.listEndpoints
-        # - com.datadoghq.kubernetes.core.listEvent
-        # - com.datadoghq.kubernetes.core.listLimitRange
-        # - com.datadoghq.kubernetes.core.listNamespace
-        # - com.datadoghq.kubernetes.core.listNode
-        # - com.datadoghq.kubernetes.core.listPersistentVolume
-        # - com.datadoghq.kubernetes.core.listPersistentVolumeClaim
-        # - com.datadoghq.kubernetes.core.listPodTemplate
-        # - com.datadoghq.kubernetes.core.listReplicationController
-        # - com.datadoghq.kubernetes.core.listResourceQuota
-        # - com.datadoghq.kubernetes.core.listService
-        # - com.datadoghq.kubernetes.core.listServiceAccount
-        # - com.datadoghq.kubernetes.core.updatePod
-        # - com.datadoghq.postgresql.select
-        # - com.datadoghq.temporal.getWorkflowResult
-        # - com.datadoghq.temporal.listWorkflows
-        # - com.datadoghq.temporal.runWorkflow
+      actionsAllowlist: []
+    # -- List of kubernetes actions to enable and for which to provide permissions (use either this or kubernetesPermissions and config.actionsAllowlist)
+    kubernetesActions:
```

Comment:
It may be helpful to be a bit more explicit here and expand on the details of `kubernetesPermissions` and `actionsAllowlist` being mutually exclusive with `kubernetesActions`, specifically with the detail that RBAC and config in PAR is populated by configuration derived from `kubernetesActions`/

Reply:
I changed the doc to ```# -- Kubernetes permissions to provide in addition to the one that will be inferred from `kubernetesActions` (useful for customObjects)```